### DISCUSSION TITLE
What's HTTP code zero means?

### DISCUSSION THREAD

**Discussion by favoyang** [upptime/upptime#119] [Q&A] [answered]
I got messages with HTTP code 0 and response time 0.
```
[CN] Unity Registry /package-name (https://packages.unity.cn/com.unity.addressables) was down:

HTTP code: 0
Response time: 0 ms
```

The server is located in China, I expect the connection from the US to China can be bumpy. So what the messages mean?
- the URL is not reachable (TCP connection failed)
- or the request is timeout?

**Comment by AnandChowdhary** [comment, marked as answer]
You're right, this means that the URL is not reachable. If the response time and HTTP code are both `0`, the connection couldn't be established.

Most likely, this is because that URL (which works perfectly fine for me) is not accessible to GitHub's IP address in the United States.

**Reply by favoyang** [reply]
Thank you.
